README.md

```diff
@@ -3,3 +3,27 @@
 A build bot
 
 [![Circle CI](https://circleci.com/gh/dkubb/bot.svg?style=shield)](https://circleci.com/gh/dkubb/bot)
+[![Deploy](https://www.herokucdn.com/deploy/button.png)](https://heroku.com/deploy)
+
+## Getting Started
+
+1. Click on Deploy to Heroku button.
+2. Remember the application name.
+3. Make sure you have AWS read/write credentials for S3.
+4. Create an S3 bucket for halcyon assets.
+5. Run the following commands:
+
+```bash
+# Clone the repository
+git clone https://github.com/dkubb/bot.git
+cd bot
+
+# Setup the local environment
+./sbin/setup.sh
+
+# Build the system locally
```

Comment:
@mbj I couldn't get halcyon to work with anything other than `/app` as the base directory. They must have some of the paths hard-coded in that diretory, or logic that is untested with different paths.

Even still I removed the hard-coded occurrences of `/app` and replaced them with a single configuration variable in the generated `env.sh` file.